PluginsAndFeatures/azure-toolkit-for-intellij/.idea/gradle.xml

```diff
@@ -14,6 +14,7 @@
             <option value="$PROJECT_DIR$/azure-intellij-plugin-appservice" />
             <option value="$PROJECT_DIR$/azure-intellij-plugin-appservice-java" />
             <option value="$PROJECT_DIR$/azure-intellij-plugin-arm" />
+            <option value="$PROJECT_DIR$/azure-intellij-plugin-azure-sdk" />
```

Comment:
We should add the `/.idea` folder to our `.gitignore` file to avoid seeing it in these PRs.